src/models/appState.ts

```diff
@@ -14,5 +15,6 @@ export type AppState = {
     peers: string[],
     favorites: string[],
     prices: Prices,
-    errors: Error
+    errors: Error,
```

Comment:
The name-type signature is a little weird here. `errors: Error` - is it singular, or plural?

Not asking to fix in this PR, but you can use GitHub issues to keep a tracker of what needs doing / investigating.

Reply:
raised as an issue and will make sure to go back and select a more appropriate name.